apps/wizdm/src/app/navigator/actionbar/action.component.ts

```diff
@@ -23,9 +23,14 @@ export class ActionComponent {
   @Input() color: ThemePalette;
 
   /** Disables the action */
-  @Input('disabled') set disableAction(value: boolean) { this.disabled = coerceBooleanProperty(value); }
+  @Input('disabled') set disableAction(value: any) { this.disabled = coerceBooleanProperty(value); }
   public disabled = false;
 
-  /** Emits when clicked */
-  @Output() activate = new EventEmitter<MouseEvent>();
+  @HostBinding('style.pointer-events') get isActionDisabled(): string { return this.disabled ? 'none' : ''; }
+
+  public onClick(event: MouseEvent) {
+
+    // Notifies teh actionbar about the action that has been activated
+    this.actionbar.activate(this);
```

Comment:
@lasfrancisco I've restored `actionbar.activate()`. I'm not sure if it adds any value though... If it's not needed button listener could be removed all together...